Hook into vault

djenriquez opened the pull request.

djenriquez committed:
Add dockerfile

djenriquez committed:
Add request to vault API, use accessToken

djenriquez committed:
Attempt to add express

djenriquez committed:
Spellcheck

djenriquez committed:
Clean up

Lucretius commented on server.js:
```diff
@@ -0,0 +1,29 @@
+    next();
+});
+
+app.listen(PORT, () => {
```
I usually throw this line on the very bottom after I register all of my routes.

Lucretius commented on server.js:
```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import favicon from 'serve-favicon';
```
probably dont need this favicon thing.

Lucretius commented on package.json:
```diff
@@ -34,12 +34,15 @@
     "react-router": "^3.0.0",
-    "react-tap-event-plugin": "^1.0.0"
+    "react-tap-event-plugin": "^1.0.0",
+    "serve-favicon": "^2.3.0"
```
same here about the favicon

Commit:
Hooking into Vault

- Using express backend server to make calls to the Vault API to bypass
CORs
- Adding login, list secrets, and get secrets calls

Commit:
Fixing conflict
